microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Sign-up for the Ad Selection API
+description: Sign-up for the Ad Selection API, which provides user-relevant ads on your site without using third-party cookies.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 10/05/2024
+---
+# Sign-up for the Ad Selection API
+<!-- https://go.microsoft.com/fwlink/?linkid=2289906 -->
+
+The Ad Selection API provides user-relevant ads on your site without using third-party cookies.  See [Ad Selection Overview](https://github.com/WICG/privacy-preserving-ads/blob/main/Ad%20Selection%20Overview.md) in the "WICG / privacy-preserving-ads" repo.
+
+The Ad Selection API can be used by:
+* **Sellers** – Supply-side platforms (SSPs), who can use the API to provide higher-quality ads for their publishers.
+* **Buyers** – Demand-side platforms (DSPs), who can use the API to improve the effectiveness of advertising campaigns.
+
+To get started using the Ad Selection API and test out the end-to-end flow, sign up for the limited Preview of the Ad Selection API.
+
+Sellers (SSPs) and buyers (DSPs) can use this guide to:
```

Comment:
```suggestion
Sellers and buyers can use this guide to:
```
already introduced the acronyms two paragraphs above, seems distracting to re-define so soon.
it would be ok to re-define one time at top of each h2 section, if it's this important to reiterate.

Reply:
result:
```
The Ad Selection API can be used by:
* **Sellers** – Operators of a supply-side platform (SSP), who can use the API to provide higher-quality ads for their publishers.
* **Buyers** – Operators of a demand-side platform (DSP), who can use the API to improve the effectiveness of advertising campaigns.
...
As a seller operating a supply-side platform or a buyer operating a demand-side platform, use this guide to:
* Test the Ad Selection API on your site.
```